src/Compilers/CSharp/Test/Emit3/FieldKeywordTests.cs

```diff
@@ -10464,6 +10464,40 @@ .locals init (System.Runtime.CompilerServices.DefaultInterpolatedStringHandler V
             VerifyMergedProperties(actualProperties, actualFields);
         }
 
+        [WorkItem("https://github.com/dotnet/roslyn/issues/75893")]
+        [Theory]
+        [CombinatorialData]
+        public void SpeculativeSemanticModel(bool includeLocal)
```

Comment:
>SpeculativeSemanticModel [](http://example.com/codeflow?start=20&length=24)

Consider adding a test for a success scenario (speculating with `field` works in a field-backed property), if we don't have one already.